src/bundler.ts

```diff
@@ -267,12 +282,13 @@ class Bundler {
     // should be removed from the document.
     const importMap: Map<string, Import|null> = new Map();
     analyzedRoot.getByKind('import').forEach((i) => importMap.set(i.url, i));
-    this.excludes.forEach((u) => {
-      if (importMap.has(u)) {
+    importMap.forEach((_, u) => {
```

Comment:
It might make more sense to only allow excludes specified as root-relative paths, which would let us do O(1) lookups.